draft-ietf-rats-architecture.md

```diff
@@ -697,6 +697,25 @@ for by hardware or by ROM code, especially if such hardware is
 physically resistant to hardware tampering.  The component that is
 implicitly trusted is often referred to as a Root of Trust.
 
+Implicit trust can also be tied to the communications link over which
+the Evidence is conveyed. As an example, the Attester and Verifier can
+communicate over a link that is established using a Root of Trust.  An example of such communications
+is over a cellular link, where the link is established by the Attester leveraging an enclave (e.g., SIM card).  The Verifier may have established the communications link and verified
+the presence of the Root-of-Trust in doing so, or may have received evidence that the communications
+link is anchored to a Root of Trust from an entity that established the
+communications link
+or may have received evidence that the communications link is anchored to a
+Root of Trust from an entity that established the communications link.
+In the latter case, anchoring the communications link to a Root of Trust would mean
+that this entity only established a communications link with the attesting
+device after verifying that the device had a Root of Trust.
```

Comment:
I don't think this wording is correct.   Maybe "a Root of Trust that this entity trusts"?

Reply:
It is at least verbose.